create_script.js

```diff
@@ -307,34 +283,22 @@ function onRepositoryChanged() {
     }
 }
 
-/// See https://stackoverflow.com/questions/105034/create-guid-uuid-in-javascript
-function uuidv4() {
-    return ([1e7]+-1e3+-4e3+-8e3+-1e11).replace(/[018]/g, c =>
-      (c ^ crypto.getRandomValues(new Uint8Array(1))[0] & 15 >> c / 4).toString(16)
-    );
-  }
-
 function generate_script() {
-    var script = "";
-
-    var projectName = document.getElementById("project_name_text_field").value.replace(/\s/g, '');
-    var projectUrl = document.getElementById("git_repository_text_field").value;
-    var enabledLibs = getEnabledLibraries();
+    const projectName = document.getElementById("project_name_text_field").value.replace(/\s/g, '');
+    const githubUsername = document.location.host.split(".")[0];
```

Comment:
instead of githubUsername I'd rather have the entire URL which makes local development easier. I prefer not having to wait for github to update the page to see my changes.